### DISCUSSION TITLE
Inttelij

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1608] [Q&A] [answered]
![Screenshot 2023-11-16 101646](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/150101c6-c33d-4b09-b879-30c8caa8d4d3)
Intellij setting changed option settled bottom how can i set  them top.

**Comment by sreedharm07** [comment, marked as answer]
Hi @palsachennaiah 
click here
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/2445e179-4ab5-415b-8223-650faf7ff95e)

**Reply by palsachennaiah** [reply]
Ok got it, thank you
